### PR TITLE
Convert RadioCal to Qml

### PR DESCRIPTION
**Pull request by DonLakeFlyer**
This is the last Setup panel to go to Qml. Finishing this up will allow me to convert the entire Setup view to Qml only, instead of the hybrid widget+Qml that it is now. This will then allow me to fix some other setup usability problems.

At this stage RadioCal unit test is turned off. I'm pushing up now to see if I can get some help testing. RadioCal is fairly involved. So extra eyes are appreciated.

RadioCal works just like it did before as far as the sequence of steps.

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
I'll continue on with getting the unit test working before I merge this. Changing from QWidget unit testing to Qml unit test is somewhat involved.

**Comment by DonLakeFlyer** [comment]
Turned out to be easier to covert RadioCal unit test than I thought.

File: src/AutoPilotPlugins/PX4/RadioComponent.cc [hunk 1]
```diff
@@ -140,7 +140,11 @@ QStringList RadioComponent::paramFilterList(void) const
 
 QWidget* RadioComponent::setupWidget(void) const
 {
-    return new PX4RCCalibration;
+    QGCQmlWidgetHolder* holder = new QGCQmlWidgetHolder();
```
Comment (vooon) [inline src/AutoPilotPlugins/PX4/RadioComponent.cc]: @DonLakeFlyer may i ask you, why you don't use C++11 `auto`? I think that is very handy, and i widely use it in mavros.
Only one thing, seems that compiling is slower, over one minute for such a small project.

Comment (DonLakeFlyer) [inline src/AutoPilotPlugins/PX4/RadioComponent.cc]: I think in general auto hides too much of the details of the code. So I stay away from it.

Comment (DonLakeFlyer) [inline src/AutoPilotPlugins/PX4/RadioComponent.cc]: > Only one thing, seems that compiling is slower, over one minute for such a small project.

I haven't noticed much compile time increase. TeamCity is reporting the same builds time for the last couple weeks.

**Pull request merged**